src/Eyeball.jl

```diff
@@ -162,6 +162,12 @@ function eye(x = Main, depth = 10; interactive = true, all = false)
             sobj = String(take!(io.io))
             _pager(node.data.value)
             resetterm()
+        elseif i == Int('S')
+            node = FoldingTrees.setcurrent!(menu, menu.cursoridx)
+            length(methods(sort, (typeof(node.data.value),)))==0 && return false
```

Comment:
Is this the best way to check if an object can be sorted? It may be. It just looks clunky.

Reply:
the only other way i can think of is a try-catch block, but that's even worse.  happy to make a change if you think of sth better.

Reply:
How about `hasmethod`?

Reply:
changed!